index.js

```diff
@@ -189,3 +210,37 @@ function addStartupSupportabilities(agent) {
     }
   }
 }
+
+/**
+ * Used to determine how the agent is getting loaded:
+ *  1. -r newrelic
+ *  2. --loader newrelic/esm-loader.mjs
+ *  3. require('newrelic')
+ *
+ *  Then a supportability metric is loaded to decide.
+ *  Note: We already take care of scenario #2 in newrelic/esm-loader.mjs
+ *
+ * @param {Agent} agent active NR agent
+ */
+function recordLoaderMetric(agent) {
+  const dashR = process.execArgv.indexOf('-r')
+  const isESM = agent.metrics.getMetric(NAMES.FEATURES.ESM.LOADER)
+
+  if (process.execArgv[dashR + 1] === 'newrelic') {
+    agent.metrics.getOrCreateMetric(NAMES.FEATURES.CJS.PRELOAD).incrementCallCount()
+  } else if (!isESM) {
+    agent.metrics.getOrCreateMetric(NAMES.FEATURES.CJS.REQUIRE).incrementCallCount()
+  }
+}
+
+/**
+ * Checks to see if `--enable-source-maps` is being used and logs a supportability metric.
+ *
+ * @param {Agent} agent active NR agent
+ */
+function recordSourceMapMetric(agent) {
+  const isSourceMapsEnabled = process.execArgv.indexOf('--enable-source-maps')
+  if (isSourceMapsEnabled > -1) {
```

Comment:
Since you don't actually need the value of the index, can we switch this to something like

```
const isSourceMapsEnabled = process.execArgv.includes('--enable-source-maps')
if (isSourceMapsEnabled) {
...
```

Just think it reads a bit easier, I understand that syntax-wise they're equivalent